pyproject.toml

```
@@ -28,17 +28,19 @@ classifiers=[
 [tool.poetry.dependencies]
 python = ">=3.10,<=3.11.8"
 rich="^11.2.0"
-matplotlib="^3.7.4"
+matplotlib="^3.9.0"
 pyyaml="^6.0.1"
 numpy="^1.21.5"
 pandas="^1.4.1"
-ultranest="3.6.4"
+ultranest="*"
```

```suggestion
ultranest="3.6.4"
```
I recall also here we had some issue with other versions, 
did you check that it works with newer?